scripts/mosh.pl

```diff
@@ -382,6 +382,11 @@ sub predict_check {
   if ( scalar @command > 0 ) {
     push @server, '--', @command;
   }
+  
+  if ( $server cmp 'mosh-server' ) {
```

Comment:
You are right I could having been using a wrapper on the server side. Problem solved! Thanks.